tests/integration/kubernetes/tests_common.sh

```diff
@@ -48,6 +48,14 @@ setup_common() {
 	node=$(get_one_kata_node)
 	[ -n "$node" ]
 	node_start_time=$(exec_host "$node" date +\"%Y-%m-%d %H:%M:%S\")
+	# If node_start_time is empty, try again 3 times with a 5 seconds sleep between each try.
```

Comment:
Hi @BbolroC !

This re-try mechanism really made the code strongest, thanks!

Any idea way the `exec_host` failed to get the time? Did it leave some `node-debugger` pods running or it at least cleaned up correctly?

Reply:
yeah, `node-debugger` was successfully launched and terminated as normal, but the script failed to fetch the info.

Reply:
ok, I was worried about it leaking debug pods on the system. So we are fine here \o/